### ISSUE TITLE
Suppor

### ISSUE THREAD

**Issue by TheJetOU** [Zarel/Pokemon-Showdown-Client#1348]

**Comment by Zarel** [comment]
> * The server will _always_ send join / leave messages

It already does though.

**Comment by Zarel** [comment]
I don't mind this change, but I think the default auto-messages could stand to be a bit smarter. I've always thought about "auto-off when usercount goes above 6" or something like that.

**Comment by TheJetOU** [comment]
> It already does though.

Yeah, I wasn't aware it just sent it as `|j|`, which I guess implies it as more "silent".

**Comment by scheibo** [comment]
`|J|` is the silent one (I made this mistake as well)

**Comment by TheJetOU** [comment]
#1352 is a phase one of sorts. I still want to eventually do:
- Add a `/force(show|hide)joins` in the future, which would show _all_ joins / leave messages
- Add a "Show Joins / leave messages" option to replays 
- Make the auto-messages smarter as @Zarel said.

**Comment by Zarel** [comment]
I don't think there needs to be a `/forcehidejoins`. I'm fine with just changing how `/hidejoins` works, and adding an `/autoshowjoins`.

**Comment by TheJetOU** [comment]
> (I also don't know what /forceshowjoins would do, but I assume you just didn't think about it when you typed that.)

It would just show `|J|` (silent) messages in the logs.

**Comment by Zarel** [comment]
> It would just show `|J|` (silent) messages in the logs.

That's what `/showjoins` does. I'm saying I don't know what `/forceshowjoins` would do differently.

**Comment by TheJetOU** [comment]
> I don't think there needs to be a /forcehidejoins. I'm fine with just changing how /hidejoins works, and adding an /autoshowjoins.

Either works for me.

> That's what `/showjoins` does. I'm saying I don't know what `/forceshowjoins` would do differently.

Ooh, I wasn't aware that was the case.

**Comment by Zarel** [comment]
...What did you think `/showjoins` did?

**Comment by joebangara43903** [comment]
![image](https://user-images.githubusercontent.com/131228993/232965543-0e41467f-a485-4c5a-86a0-7cab26c74364.png)
